lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -133,4 +134,21 @@ export abstract class ClientApplication {
     readCache(): JsonCache {
         return Serializer.serializeAllCache(this.storage.getCache());
     }
+
+    async readFromProvidedCache(): Promise<void> {
```

Comment:
Developers call this to initiate read and write from cache using the cache plugin